content/1.documentation/2.clients/1.cli.md

```diff
@@ -49,18 +49,41 @@ hopp test -e environment.json kitchen-sink-hoppscotch-collection.json
 
 Hoppscotch allows templates in several places. For example, you could specify your endpoint URL as `<<baseurl>>/post` and specify baseurl as [`https://echo.hoppscotch.io`](https://echo.hoppscotch.io) in an environment file.
 
-The expected format of an environment file is:
+Hoppscotch CLI support environment files in two specific formats:
+
+### 1. Multiple Environment Variables Export Format
+
+This format is generated by Hoppscotch when you export any of your environment. It includes a named environment (nameOfEnv) with multiple key-value pairs, allowing you to define various variables within a single file.
+  ```json
+  {
+    "nameOfEnv": "my_env",
+    "variables": [
+      {
+        "key1": "base_url",
+        "value1": "https://echo.hoppscotch.io"
+      },
+      {
+        "key2": "auth_token",
+        "value2": "xxxxxxxxxxxx"
+      }
+    ]
+  }
+```
+### 2. Single Environment Variable Export Format
```

Comment:
```suggestion

### 2. Single Environment Variable Export Format
```

Reply:
This is not an apt name for this format. This format is just an object with environments written in key value pairs. You can maybe call it a legacy format as this is not how the app exports the environment entries.